sdcm/sct_config.py

```diff
@@ -672,6 +672,8 @@ class SCTConfiguration(dict):
              help=""),
         dict(name="test_sst3", env="SCT_TEST_SST3",  type=boolean,
              help=""),
+        dict(name="enable_authorization_after_upgrade", env="SCT_ENABLE_AUTHORIZATION_AFTER_UPGRADE",  type=boolean,
```

Comment:
why not split it into two ? making it future proof, and enabling other Authorizers ?
```yaml
authorization_in_upgrade: 'ScyllaAuthorizer'
remove_authorization_in_rollback: true
```

Reply:
fixed.